java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -5160,6 +5160,58 @@ public HostColumnVector copyToHost() {
     }
   }
 
+  /**
+   * Calculate the total space required to copy the data to the host.
+   */
+  public long getHostBytesRequired() {
+    return getHostBytesRequiredHelper(this);
+  }
+
+  /*
+   * Align given size to account for host allocation alignment of 64
+   */
+  private static long alignAllocSize(long size) {
+    final long align = 64; // must be a power of two
```

Comment:
nit: a power of two requirement can be emphasized
with a `1 << 6` or a binary literal `0B1000000`

Where is 64 coming from? Is there a mnemonic constant for this?  Can this be represented as some DTypeEnum.<type>64.sizeInBytes * 8?